### ISSUE TITLE
LUA Errors!

### ISSUE THREAD

**Issue by GregoryTwix** [tgp1994/LiquidRP#1]
[ERROR] stack overflow
  1. __newindex - [C]:-1
   2. SetFont - lua/vgui/dlabel.lua:53
    3. ApplySchemeSettings - gamemodes/liquiddrp/gamemode/fadmin/cl_interface/cl_controls.lua:37
     4. SetFont - lua/vgui/dlabel.lua:55
      5. ApplySchemeSettings - gamemodes/liquiddrp/gamemode/fadmin/cl_interface/cl_controls.lua:37
       6. SetFont - lua/vgui/dlabel.lua:55
        7. ApplySchemeSettings - gamemodes/liquiddrp/gamemode/fadmin/cl_interface/cl_controls.lua:37
         8. SetFont - lua/vgui/dlabel.lua:55
          9. ApplySchemeSettings - gamemodes/liquiddrp/gamemode/fadmin/cl_interface/cl_controls.lua:37
           10. SetFont - lua/vgui/dlabel.lua:55
            11. ApplySchemeSettings - gamemodes/liquiddrp/gamemode/fadmin/cl_interface/cl_controls.lua:37
             12. SetFont - lua/vgui/dlabel.lua:55
              13. ApplySchemeSettings - gamemodes/liquiddrp/gamemode/fadmin/cl_interface/cl_controls.lua:37
               14. SetFont - lua/vgui/dlabel.lua:55
                15. ApplySchemeSettings - gamemodes/liquiddrp/gamemode/fadmin/cl_interface/cl_controls.lua:37
                 16. SetFont - lua/vgui/dlabel.lua:55

**Comment by Zenternal** [comment]
Right. FAdmin is the only thing that actually loaded. So whats the simple solution to this? Delete FAdmin. To do so, don't delete the fadmin folder, however, go to `gamemode/liquidrp/sh_liquidrp.lua` in the gamemode's base folder, at the top of the file set `UseFadmin` to `false`


Sorry this is extremely late, but for anyone else getting the issue.

**Comment by 0xym0ron** [comment]
+rep fast response
